test/core/store.test.js

```diff
@@ -33,7 +33,7 @@ describe('Core.Store', function () {
 				new Store(null);
 			});
 			proclaim.throws(function() {
-				new Store(function(){});
+				new Store(function(){ /* empty */ });
```

Comment:
bit of javascript trivia: for a no-op you can pass `Function.prototype` as a callback